<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>ChatMessageWidget</class>
    <widget class="QWidget" name="ChatMessageWidget">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>400</width>
                <height>50</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Form</string>
        </property>
        <property name="styleSheet">
            <string notr="true">/* Ensure the base widget is transparent */
                QWidget {
                background-color: transparent;
                }

                QTextEdit {
                border: 1px solid #444;
                border-radius: 8px;
                padding: 8px;
                font-size: 14px;
                /* We'll set the background color in the .py file */
                }
            </string>
        </property>
        <layout class="QHBoxLayout" name="mainLayout">
            <property name="leftMargin">
                <number>0</number>
            </property>
            <property name="topMargin">
                <number>2</number>
            </property>
            <property name="rightMargin">
                <number>0</number>
            </property>
            <property name="bottomMargin">
                <number>2</number>
            </property>
            <item>
                <spacer name="leftSpacer">
                    <property name="orientation">
                        <enum>Qt::Orientation::Horizontal</enum>
                    </property>
                    <property name="sizeType">
                        <enum>QSizePolicy::Expanding</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                        <size>
                            <width>10</width>
                            <height>20</height>
                        </size>
                    </property>
                </spacer>
            </item>
            <item>
                <widget class="QTextEdit" name="messageContent">
                    <property name="sizePolicy">
                        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                            <horstretch>0</horstretch>
                            <verstretch>0</verstretch>
                        </sizepolicy>
                    </property>
                    <property name="verticalScrollBarPolicy">
                        <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
                    </property>
                    <property name="horizontalScrollBarPolicy">
                        <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
                    </property>
                </widget>
            </item>
            <item>
                <spacer name="rightSpacer">
                    <property name="orientation">
                        <enum>Qt::Orientation::Horizontal</enum>
                    </property>
                    <property name="sizeType">
                        <enum>QSizePolicy::Expanding</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                        <size>
                            <width>10</width>
                            <height>20</height>
                        </size>
                    </property>
                </spacer>
            </item>
        </layout>
    </widget>
    <resources/>
    <connections/>
</ui>

